src/__tests__/render.js

```diff
@@ -90,3 +90,24 @@ it('supports fragments', () => {
   cleanup()
   expect(document.body.innerHTML).toBe('')
 })
+
+test('renders options.wrapper around node', () => {
+  const WrapperComponent = ({children}) => (
+    <div data-testid="wrapper">{children}</div>
+  )
+
+  const {container, getByTestId} = render(<div data-testid="inner" />, {
+    wrapper: WrapperComponent,
```

Comment:
Should this be: `wrapper: ({children}) => <Wrapper>{children}</Wrapper>` so people have more flexibility?

Reply:
You mean call it as a function instead of with createElement?